api/src/identity-access-management/domain/usecases/create-user.usecase.js

```diff
@@ -69,9 +67,8 @@ const createUser = async function ({
       }
     }
 
-    const token = crypto.randomUUID();
-    await emailValidationDemandRepository.save({ token, userId: savedUser.id });
-    await mailService.sendAccountCreationEmail(savedUser.email, localeFromHeader, redirectionUrl);
+    const token = await emailValidationDemandRepository.save(savedUser.id);
+    await mailService.sendAccountCreationEmail(savedUser.email, localeFromHeader, token, redirectionUrl);
```

Comment:
Changer la signature de la méthode pour utiliser un objet au lieu de 4 paramètres

```suggestion
    await mailService.sendAccountCreationEmail({email: savedUser.email, localeFromHeader, token, redirectionUrl});
```